RELEASING.md

```diff
@@ -71,27 +71,10 @@ token](https://help.github.com/articles/creating-a-personal-access-token-for-the
     $ git checkout -b v$MAJOR.$MINOR.x master
     $ git push upstream v$MAJOR.$MINOR.x
     ```
+    The branch will be automatically protected by the GitHub branch protection rule for release
+    branches.
 
-2. Enable branch protection for the new branch, if you have admin access.
-   Otherwise, let someone with admin access know that there is a new release
-   branch.
-
-    - Open the branch protection settings for the new branch, by following
-      [Github's instructions](https://help.github.com/articles/configuring-protected-branches/).
-    - Copy the settings from a previous branch, i.e., check
-      - `Protect this branch`
-      - `Require pull request reviews before merging`
-      - `Require status checks to pass before merging`
-      - `Include administrators`
-
-      Enable the following required status checks:
```

Comment:
Will these CI checks be automatically enabled as well?

Reply:
Yes, currently there is one branch protection rule for all release branches, which enables the same checks as the per-branch branch protection settings that we had before.